python/cugraph/cugraph/community/louvain.py

```diff
@@ -112,6 +127,9 @@ def louvain(G, max_level=None, max_iter=None, resolution=1.0, threshold=1e-7):
     if max_level is None:
         max_level = 100
 
+    if max_level > 1000:
+        max_level = 1000
```

Comment:
You need to at least update the docstring to say that you're changing the user's option.  A better change would be to *also* omit a warning, but the docstring must be updated at a minimum.

Reply:
Updated